### ISSUE TITLE
doesn't overriding operators 

### ISSUE THREAD

**Issue by kartikag01** [react-querybuilder/react-querybuilder#25]
i am using latest version of `"react-querybuilder": "^1.3.4"` and i override operators 

```
const operators = [
  { name:'null', label:'Is Null' },
  { name:'notNull', label:'Is Not Null' },
  { name:'in', label:'In' },
  { name:'notIn', label:'Not In' }
];

```

```
 <QueryBuilder fields={fields}
                      operators={operators}
                      combinators={combinators}
                      controlElements={controlElements}
                      controlClassnames={{ fields:'form-control' }}
                      onQueryChange={this.logQuery.bind(this)}/>
```

but still its showing default operators.

<img src="http://chatteron.io/images/1/ryZCD2avl.png" />

**Comment by pavanpodila** [comment]
Thanks for reporting @KARTIK01. Definitely a bug in the way the `getOperators()` is implemented

**Comment by kartikag01** [comment]
@pavanpodila 
```
  getOperators(field) {
    if (this.props.operators) {
      return this.props.operators;
    }
    return QueryBuilder.defaultOperators;
  }
```

i suppose this should be correct `getOperators` ?

**Comment by pavanpodila** [comment]
This should be fixed in v1.3.5 as part of this [commit](https://github.com/sapientglobalmarkets/react-querybuilder/commit/064687abfeb2c3135806744d565a7e0d4778abae)

Please verify

**Comment by pavanpodila** [comment]
Closing it for now as it works as expected.
